### DISCUSSION TITLE
Where to find the windows installer

### DISCUSSION THREAD

**Discussion by IceflowRE** [ImageMagick/ImageMagick#2974] [General]
The downlaod page on the website remains empty and the releases only have the AppImage formats for Linux.

**Comment by snibgo** [comment]
When a webpage has a problem, please give the URL you are trying to access.

The usual page for Windows downloads is https://www.imagemagick.org/script/binary-releases.php#windows . It worked a few days ago, but currently says "File not found." I suppose this is a temporary problem.

**Comment by fmw42** [comment]
I can confirm that the binary download page at https://imagemagick.org/script/download.php is empty apart from the advertisement at the top
